{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": []
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "security@golang.org",
      "ID": "CVE-2025-47907"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "Cancelling a query (e.g. by cancelling the context passed to one of the query methods) during a call to the Scan method of the returned Rows can result in unexpected results if other queries are being made in parallel. This can result in a race condition that may overwrite the expected results with those of another query, causing the call to Scan to return either unexpected results from the other query or an error."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": []
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://go.dev/cl/693735",
          "refsource": "",
          "tags": [],
          "url": "https://go.dev/cl/693735"
        },
        {
          "name": "https://go.dev/issue/74831",
          "refsource": "",
          "tags": [],
          "url": "https://go.dev/issue/74831"
        },
        {
          "name": "https://groups.google.com/g/golang-announce/c/x5MKroML2yM",
          "refsource": "",
          "tags": [],
          "url": "https://groups.google.com/g/golang-announce/c/x5MKroML2yM"
        },
        {
          "name": "https://pkg.go.dev/vuln/GO-2025-3849",
          "refsource": "",
          "tags": [],
          "url": "https://pkg.go.dev/vuln/GO-2025-3849"
        }
      ]
    }
  },
  "impact": {},
  "lastModifiedDate": "2025-08-07T21:26Z",
  "publishedDate": "2025-08-07T16:15Z"
}